webpack.config.js

```diff
@@ -5,17 +5,28 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.js$/,
+        test: /\.jsx$/,
         exclude: /node_modules/,
         loader: 'babel-loader',
       },
+      {
+        test: /\.svg$/,
```

Comment:
You don't need this loader. Create a react components for icons

Reply:
I have to create one component, that will receive src and alt  in props. Yes?

Reply:
Nope, just a react component with svg in render. Thats it.